test/test_mkldnn.py

```diff
@@ -567,6 +567,24 @@ def test_gelu(self):
         self.assertEqual(y1, y2)
         self.assertEqual(x1.grad, x2.grad.to_dense())
 
+    def test_gelu_transpose(self):
+        m = torch.nn.GELU()
+        x = torch.randn((4, 6, 6), dtype=torch.float32) * 10
+        y_target = torch.randn((4, 6, 6), dtype=torch.float32) * 10
+        x1 = x.clone().requires_grad_(True)
+        x2 = x.clone().requires_grad_(True)
+        y1 = m(x1).transpose(-1, -2)
```

Comment:
How are you sure that mkldnn is being used here?

Reply:
In https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/Activation.cpp#L377-L408, use_mkldnn is True and approximate_type is default to None, mkldnn will be used here.